erts/emulator/beam/erl_db.c

```diff
@@ -2663,13 +2663,51 @@ BIF_RETTYPE ets_lookup_element_3(BIF_ALIST_3)
     case DB_ERROR_BADKEY:
         BIF_P->fvalue = EXI_BAD_KEY;
         BIF_ERROR(BIF_P, BADARG | EXF_HAS_EXT_INFO);
+    // TODO: error message for the DB_ERROR_BADITEM case
```

Comment:
Either fix the "TODO" or remove it. The error formatting seems to work without it.